www/js/controller.js

```diff
@@ -865,7 +865,7 @@ define([
 				console.log("Parser error: ", err);
 				console.log(err.message);
 				console.log(err.Error);
-				this.setAuthorStatus(this.currentID,'equation','incorrect');
+				this._model.authored.setAuthorStatus(this.currentID,'equation','incorrect');
```

Comment:
this should be active whenever it is used in controller. Because this code will use authored status which we do not want. If possible make this change whereever needed that way it would be easier for us when we work on student mode.